lenses/models/datamodel.py

```diff
@@ -13,24 +13,33 @@
 from pprint import pprint
 
 from . import SingleObject, Lenses
-
+from mysite.language_check import validate_language
+from mysite.image_check import validate_image_size
 
 
 class GenericImage(SingleObject,DirtyFieldsMixin):
     lens = models.ForeignKey(Lenses,
                              null=True,
                              on_delete=models.SET_NULL,
                              related_name="%(class)s")
+
+    # validate language for this one: likely english check will fail if weird filename.
+    # not including it.
+    # language check skipped for this one until more relaxed english check
```

Comment:
This is not a file name, just a descriptive name for the upload generic image, e.g. 'Field of view of 40''", or it could even be a plot... So better to check it for bad language.